Utilities/Maintenance/UpdateITKGitTagPythonSetup.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+program_name=$0
+
+function usage {
+  echo "Usage: ${program_name} "
+  echo "  param1 Setup.py file path"
+  exit 1
+}
+
+# Display usage
+if [ ${#@} != 1 ]; then
+  usage
+fi
+
+filename=$1
```

Comment:
I guess that same remark from remote update applies here regarding relative path to `Setup.py` file.

Reply:
Thanks @dzenanz . Answering in #641.